nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -31,18 +31,42 @@ def get_all_aliases(cls) -> List[str]:
         return cls.op_names
 
 
+@dataclass
+class OpWeightDef:
+    """
+    Contains the information about the weight of the operation.
+
+    :param weight_channel_axis: Axis for weight per-channel quantization, meaning the number of output filters.
```

Comment:
I would suggest using the plural term, like `weights` and `activations` everywhere because this is what we use to describe the DL model content, **not** `weight and activation of the model`.